Content.Server/Tiles/FloorTileSystem.cs

```diff
@@ -49,10 +50,25 @@ private void OnAfterInteract(EntityUid uid, FloorTileComponent component, AfterI
                 if (physics.TryGetComponent(ent, out var phys) &&
                     phys.BodyType == BodyType.Static &&
                     phys.Hard &&
-                    (phys.CollisionLayer & (int) CollisionGroup.Impassable) != 0)
-                    return;
+                    (phys.CollisionLayer & (int) CollisionGroup.Impassable) != 0) 
+                    {
+                        // e.g. you can place tile under directional windows
+                        var gridUid = location.GetGridUid(EntityManager);
```

Comment:
You're getting the griduid inside of the loop when it should just be done outside.